src/libmusig2.h

```diff
@@ -7,212 +7,141 @@
 #include <secp256k1_extrakeys.h>
 #include <secp256k1_schnorrsig.h>
 #include "random.h"
+#include "../config.h"
 
 
 
-/*************************************************************************/
-/****************************** DEFINITIONS ******************************/
-/*************************************************************************/
-
-
+#define V                   2          // Number of nonce values
 #define SCALAR_BYTES       32          // SCALAR BYTES
-#define XONLY_BYTES        32          // XONLY PK BYTES
-#define PK_BYTES           64          // FULL PK BYTES
-
-
-/*************************************************************************/
-/************************ FUNCTION DECLARATIONS **************************/
-/*************************************************************************/
-
-
-/** Function    : MuSig2_KeyGen
- *  Purpose     : Generates the keypair of given signer for a random secret key in given ctx.
- *  Parameters  : IN        : ctx, a secp256k1_context object.
- *                IN/OUT    : signer, secp256k1_keypair object.
+#define XONLY_BYTES        32          // XONLY PUBLIC KEY BYTES
+#define PAR_SIG_BYTES      32          // PARTIAL SIGNATURE BYTES
+#define PK_BYTES           64          // FULL SIZE PUBLIC KEY BYTES
+#define SCH_SIG_BYTES      64          // SCHNORR SIGNATURE BYTES
+
+
+/** Struct      : musig2_context
+ *  Purpose     : Stores the parameters of musig2.
+ *  Parameters  : ctx: a secp256k1_context object.
+ *              : X_: Aggregated public key.
+ *              : R_: Aggregated R.
```

Comment:
Any reason why we use this notation here? why not something more descriptive, such as `aggr_pk` or `aggr_R`?

Reply:
Yes, I think its better to change. When writing code we can be more descriptive with the variable names than when using mathematical notation. Thanks!